grafana/include/wait-for-db.sh

```diff
@@ -11,8 +10,8 @@ wait_for_it() {
 }
 
 # Parse string like: postgres://postgres:postgres@astronomer-postgresql.astronomer.svc.cluster.local:5432/astronomer_houston
-POSTGRES_HOST=$(echo "${GF_DATABASE_URL}" | awk -F@ '{print $2}' | awk -F: '{print $1}')
-POSTGRES_PORT=$(echo "${GF_DATABASE_URL}" | awk -F@ '{print $2}' | awk -F: '{print $2}' | awk -F/ '{print $1}')
+POSTGRES_HOST="$(echo "${GF_DATABASE_URL##*@}" | awk -F: '{print $1}')"
```

Comment:
hm... we use similar code in houston-api https://github.com/astronomer/houston-api/blob/main/bin/entrypoint#L27-L28 FYI

Reply:
@andriisoldatenko you should update that houston code with these change.

Reply:
Or we could make something that is more reliable in python.